jsk_recognition_utils/node_scripts/polygon_array_publisher.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
```

Comment:
In other nodes, the license is estimated by package.xml (probably BSD), but you specify copyright, and I thought it was under another license.